apps/backend/db_patches/0165_AddProposalScientistComment.sql

```diff
@@ -0,0 +1,11 @@
+DO
+$$
+BEGIN
+	IF register_patch('0165_AddProposalScientistComment.sql', 'Farai Mutambara', 'Adding proposal comment by scientist', '2024-10-31') THEN
+		BEGIN
+		ALTER table proposals ADD COLUMN comment_by_scientist TEXT NULL;
```

Comment:
Since this is the requirement exclusively for Xpress Proposals, can we not use the base table(proposal) and create a new table. This will help with not creating too much overhead on the proposal table and be a scalable approach as well. 

Let me know your thoughts.

Reply:
I have added the changes.